src/controller/game_controller.py

```diff
@@ -80,6 +80,10 @@ def playoneround(self):
                         return False
                     elif event.type == KEYDOWN and event.key == K_ESCAPE:
                         return False
+                    elif event.type == pygame.MOUSEBUTTONDOWN:
```

Comment:
Don't we want something like MOUSEBUTTONDOWN then MOUSEBUTTONUP (click-release)?

Reply:
Mmmmmh, maybe we can use MOUSEBUTTONUP only instead ?

Reply:
OK